ChecksFile.wl

```diff
@@ -179,7 +179,7 @@ Idis[data_,\[CapitalOmega]_,k_,En_,n_,xflavor_]:=Module[{cos\[Theta],\[Phi]0,\[P
 
 cos\[Theta]=data["mids"];
 {\[CapitalOmega]p,kp,\[Omega]}=IdisShifts[data,\[CapitalOmega],k,En,xflavor];
-\[CapitalOmega]minuskpcos\[Theta]=Reap[Do[Sow[IdisBottom[data,\[CapitalOmega],k,En,xflavor][[i]]],{i,1,Length[cos\[Theta]]}]][[2,1]];
+\[CapitalOmega]minuskpcos\[Theta]=IdisBottom[data,\[CapitalOmega],k,En,xflavor];
```

Comment:
This appears to be renaming Idisbottom as omegap-kpcostheta

Reply:
This is another example of how IdisBottom was being called Length(costheta) times. So in each iteration of the do loop, the full array was being constructed and then one element was being accessed. In the new version, you just calculate the array with IdisBottom (the output is unchanged) and set the the left hand side to this array. No do loop needed!